src/Shared/Recruit.Vacancies.Client/Infrastructure/Mongo/MongoDbRetryPolicy.cs

```diff
@@ -10,14 +10,25 @@ public static class MongoDbRetryPolicy
     {
         public static RetryPolicy GetRetryPolicy(ILogger logger)
         {
-            return Policy
-                .Handle<MongoException>()
+            return AddWaitAndRetry(Policy.Handle<MongoException>(), logger);
+        }
+
+        public static RetryPolicy GetConnectionRetryPolicy(ILogger logger)
```

Comment:
I'm not sure having two different policies is the way to go here.  I think you can just add this to the exisitng policy
```
.Handle<MongoException>()
.Or<MongoConnectionClosedException>()
.....
```

Reply:
MongoConncectionClosedException is descended from MongoException. 
What I want to do here is only try again for the transient error experienced when the server cannot be reached. The code itself handle the Timeout exception (no more RUs), and any other error is considered non-recoverable and aborts.

Reply:
👍